src/databricks/labs/remorph/snow/snowflake.py

```diff
@@ -490,3 +507,30 @@ def _json_column_op(self, this, path):
                 return self.expression(local_expression.Bracket, this=this, expressions=[path])
 
             return self.expression(exp.Bracket, this=this, expressions=[path])
+
+        def _parse_window(self, this: exp.Expression | None, alias: bool = False) -> exp.Expression | None:
```

Comment:
<img width="355" alt="image" src="https://github.com/user-attachments/assets/0249d663-92f8-424f-97cb-4d27289ba56f">

looks like 11 integration tests started failing because of this change. Can you add these to `com.databricks.labs.remorph.transpilers.SnowflakeToDatabricksTranspilerTest` and make sure the core tests pass?

Reply:
These failing tests are, in theory, fixed by #792. So I believe rebasing this branch on main should be enough to get a better coverage report.